### PR TITLE
Work around Salt 18089

### PR DESCRIPTION
**Pull request by justinsb**
Directly create /etc/sysctl.d/99-salt.conf

### CODE REVIEW

File: cluster/saltbase/salt/docker/init.sls [hunk 1]
```diff
@@ -43,6 +43,10 @@ docker:
     - repl: '# net.ipv4.ip_forward=0'
 {% endif %}
 
+# Work around Salt #18089
```
Comment (zmerlynn) [inline cluster/saltbase/salt/docker/init.sls]: Can you go ahead and link it? https://github.com/saltstack/salt/issues/18089

**Comment by zmerlynn** [comment]
LGTM except a nit

**Comment by justinsb** [comment]
Added link to github issue as suggested - should have done that in the first place!

**Comment by zmerlynn** [comment]
LGTM

**Comment by zmerlynn** [comment]
cc @dchen1107 @alex-mohr for rubber stamps

**Comment by alex-mohr** [comment]
LGTM

**Comment by brendandburns** [comment]
@k8s-bot test this [testing build queue, sorry for the noise]

**Comment by alex-mohr** [comment]
Sorry I missed this earlier, Justin.

**Pull request merged**